backend/app.py

```diff
@@ -1,12 +1,20 @@
 # flask_web/app.py
 
 from flask import Flask
-app = Flask(__name__)
+from flask_sqlalchemy import SQLAlchemy
+from os.path import isfile
+from pathlib import Path
+
+SQLITE_FILE = '/db/digitalesWartenDB.sqlite'
 
-@app.route('/')
-def hello_world():
-    return 'Hey, we have Flask in a Docker container!'
+app = Flask(__name__)
 
+app.config['SQLALCHEMY_DATABASE_URI'] = f'sqlite:///{SQLITE_FILE}'
+db = SQLAlchemy(app)
 
-if __name__ == '__main__':
-    app.run(debug=True, host='0.0.0.0')
+if not isfile(SQLITE_FILE):
```

Comment:
use Path.is_file() so we don't mix two path libraries

Reply:
Done